WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Shipping Label/PrintShippingLabelViewController.swift

```diff
@@ -185,21 +201,69 @@ private extension PrintShippingLabelViewController {
             configurePaperSizeOptions(cell: cell)
         case let cell as ImageAndTitleAndTextTableViewCell where row == .printingInstructions:
             configurePrintingInstructions(cell: cell)
+        case let cell as ButtonTableViewCell where row == .printButton:
+            configurePrintButtonRow(cell: cell)
+        case let cell as ButtonTableViewCell where row == .saveButton:
+            configureSaveButton(cell: cell)
         default:
             break
         }
     }
 
+    func rowsToDisplay() -> [Row] {
+        var rows: [Row?]
+        switch printType {
+        case .print:
+            rows = [
+                .spacerBetweenHeaderCells,
+                .headerText,
+                .spacerBetweenHeaderCells,
+                .headerImage,
+                .spacerBetweenHeaderCells,
+                .paperSize,
+                .spacerBetweenPaperSizeSelectorAndInfoLinks,
+                .printButton,
+                .saveButton,
+                .paperSizeOptions,
+                .printingInstructions
+            ]
+        case .reprint:
+            rows = [
+                .headerText,
+                .infoText,
+                .spacerBetweenInfoTextAndPaperSizeSelector,
+                .paperSize,
+                .spacerBetweenPaperSizeSelectorAndInfoLinks,
+                .paperSizeOptions,
+                .printingInstructions
+            ]
+        }
+        return rows.compactMap { $0 }
+    }
+
     func configureHeaderText(cell: BasicTableViewCell) {
-        cell.textLabel?.text = Localization.headerText
+        switch printType {
+        case .print:
+            cell.textLabel?.text = Localization.printHeaderText
+            cell.textLabel?.applyHeadlineStyle()
+            cell.textLabel?.textAlignment = .center
+        case .reprint:
+            cell.textLabel?.text = Localization.reprintHeaderText
+            cell.textLabel?.applyBodyStyle()
+        }
```

Comment:
nit: it would be good to have the `textAlignment` also inside the `reprint` case. The main reason is that since we are using reusable cells, the `textAlignment` can remain `.center` in the case for some reason, there is a switch between `print` and `reprint`.